README.md

```diff
@@ -2,6 +2,9 @@
 
 Create React apps with no build configuration.
 
+* [Getting Started](#getting-started)
+* [Guide](https://github.com/facebookincubator/create-react-app/blob/master/template/README.md)
```

Comment:
I think you're right. What about this:

* Getting Started – How to create a new app.
* Usage Guide – How to develop apps bootstrapped with Create React App.